Pull(-request) my weight

austintraver opened the pull request:
in this group project. Apologies that it took until
Sunday morning to get it done.

austintraver committed:
Create skeleton code for DatabaseClient

This commit implements the result of a long
night battling an inconsistent set of API
documentation.

johntannerofficial reviewed (commented):
When will you have the implementation done for the functions?

austintraver commented:
> When will you have the implementation done for the functions?

It’ll be a little easier to implement them when there are classes to create implementations for. I forget who is working on that currently. Haven’t seen any pull requests other than the two I have made so far.

charliepyle committed:
Implement User and Listing classes

austintraver committed:
Resolve module name inconsistencies

It should be com/asparagus/usclassifieds, sometimes there were
files in com/example/usclassifieds, but this commit fixes that

austintraver committed:
Support adding users and listings to database

Users and listings can be added to the database,
and can also be removed from the database. The
friend request feature has not currently been
created, as I'm not sure where we landed on whether
we're creating a social network, as this creates
some design challenges
